chrome/model.js

```diff
@@ -8,23 +8,17 @@ GmailToTrello.Model = function(parent) {
     };
     this.parent = parent;
     this.settings = {};
-    this.isInitialized = false;
+    this.isPopupDataLoaded = false;
+    this.isThreadCardsLoaded = false;
+    this.isAuthorized = false;
```

Comment:
Like I said above, init() now just does authorizes Trello. Then there are two processes now happening in this model. The first is PopupDataLoaded, which is the process that was already here. The second is ThreadCardsLoaded, which is the new process that happens immediately when the page loads to find cards associated with the current thread.